### PR TITLE
Set Portfolio Desktop version

### PR DESCRIPTION
**Pull request by algerina**
After working on the Mobile version of the Portfolio, here I worked the responsivity of this portfolio.
 I set the desktop version by adding a media query (the transition point is 900px)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,19 +12,24 @@
...
+        <ul class="desktop-toolbar">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
       </nav>
     </header>
-
-    <section class="headline-section">
+    <section class="headline-section headline-sectionds">
```
Comment (carlos-ssh) [inline index.html:27]: This Section it's overlapping your application in your X-Axis Kindly fix it.
Your live version:
Screen Shot 2021-08-27 at 20 50 39
Local Version:
Screen Shot 2021-08-27 at 20 52 35

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @algerina , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (algerina) [reply index.html:27]: Thank you for your constructive review, but I didn't really understand. Can't you explain more please.

**Review by carlos-ssh — APPROVED** [review]
Hi @algerina , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**